### DISCUSSION TITLE
postgres -nothing happens

### DISCUSSION THREAD

**Discussion by abcnow** [reflex-dev/reflex#2914] [General]
I am trying to use Postgres.. I have a connect_db.py and installed the drivers for postgres "psycopg2-binary"

from sqlmodel import create_engine

def connect():
    engine = create_engine("postgresql+psycopg2-binary://postgres:123456@localhost:5432/dbnow")
    return engine

but when I ran "reflex run" it is not creating the tables in the database and I am not getting any error.. what should I do to fix this?

**Comment by picklelo** [comment]
If you want to use Reflex' builtin database support, you don't need to do `create_engine` yourself. Just set the `db_url` in your `rxconfig.py` as shown in the [database docs](https://reflex.dev/docs/database/overview/#connecting).

It will only create the tables if you use our builtin way, not if you're creating your connections outside of reflex.
